roles/ceilometer-common/tasks/monitoring.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: ceilometer-api check
```

Comment:
Just saw other processes monitored in ceilometer-controller. Then here are for common. We have 6 processes in controllers and 1 process in computes. I guess all the monitors should be in either ceilometer-controller or ceilometer-data and nothing should be in ceilometer-common.